doc/api/stream.md

```diff
@@ -253,16 +469,16 @@ readable: null
 end
 ```
 
-#### readable.isPaused()
+##### readable.isPaused()
 
 * Return: {Boolean}
 
-This method returns whether or not the `readable` has been **explicitly**
-paused by client code (using [`stream.pause()`][stream-pause] without a
-corresponding [`stream.resume()`][stream-resume]).
+The `readable.isPaused()` method returns `true` if `readable` has been
+**explicitly** paused by client code (using the [`stream.pause()`][stream-pause]
+method without a corresponding call to [`stream.resume()`][stream-resume]).
```

Comment:
Not true. This is part of the `pipe` mechanism (https://github.com/nodejs/node/blob/master/lib/_stream_readable.js#L552-L569).